txt/BUFR_TableD_en.txt

```diff
@@ -480,6 +480,16 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 01,Location and identification sequences,301131,(Pixel geolocation),,005021,Bearing or azimuth,,,Operational
 01,Location and identification sequences,301131,(Pixel geolocation),,007025,Solar zenith angle,,,Operational
 01,Location and identification sequences,301131,(Pixel geolocation),,005022,Solar azimuth,,,Operational
+01,Location and identification sequences,301132,(Common header sequence),,301150,WIGOS Identifiers,,,Operational
+01,Location and identification sequences,301132,(Common header sequence),,301001,WMO block and station numbers,,,Operational
+01,Location and identification sequences,301132,(Common header sequence),,301021,Latitude/longitude (high accuracy),Sensor,,Operational
+01,Location and identification sequences,301132,(Common header sequence),,007030,Height of station ground above mean sea level,,,Operational
```

Comment:
Is this supposed to be part of the Common Header Sequence? 

007030,Height of station ground above mean sea level